examples/QmlBarcodeReader/qml/GeneratorPage.qml

```diff
@@ -0,0 +1,254 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtMultimedia 5.12
+import QtQuick.Layouts 1.12
+import com.scythestudio.sbarcodereader 1.0
+
+Item {
+    id: root
+    anchors.fill: parent
+
+    SBarcodeGenerator {
+        id: barcodeGenerator
+        onProcessFinished: {
+            console.log(barcodeGenerator.filePath);
+            image.source = "file:///" + barcodeGenerator.filePath
+        }
+    }
+
+    Rectangle {
+        anchors.fill: parent
+        height: parent.height
+        width: parent.width
+        visible: true
+        color: "white"
+
+        Rectangle {
+            id: inputRect
+            z: 100
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.right: parent.right
+            height: 40
+            TextField {
+                id: textField
+                placeholderText: qsTr("Input")
+                anchors.fill: parent
```

Comment:
In our conventions we have a rule that says that properties derived from Item like height, width, anchors (and much more) should be listed here, so move anchors above placeholder.